main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/companieshouse/chs.go/kafka/producer"
+	. "github.com/smartystreets/goconvey/convey"
+	"testing"
```

Comment:
Please run `goimports` against this file (set your editor to do this when saving files)

Reply:
done